addons/mail/static/src/models/message_view.js

```diff
@@ -263,6 +263,19 @@ registerModel({
             identifying: true,
             inverse: 'messageView',
         }),
+        displayInChatBubble: attr({
+            compute() {
+                if (!this.message) {
+                    return clear();
+                }
+                return (
+                    this.message.is_discussion ||
+                    this.message.is_notification ||
+                    this.message.isFromOdooBot
```

Comment:
Same question here, can't we make the message the proper type and/or subtype instead of adding this hacky condition?